magenta/models/score2perf/datagen_beam.py

```diff
@@ -42,10 +41,11 @@
 # TODO(iansimon): this should probably be defined in the problem
 SCORE_BPM = 120.0
 
-FLAGS = flags.FLAGS
 flags.DEFINE_string(
     'pipeline_options', '',
     'Command line flags to use in constructing the Beam pipeline options.')
+FLAGS = flags.FLAGS
+FLAGS(sys.argv)
```

Comment:
Why is this needed?

Reply:
without it, usage later complains that we're trying to use the flag before it's been declared for some reason, and the specs fail. but actually it looks like the `tf.app.flags.FLAGS` wrapper rather than importing from `absl`. just made that change